### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by Hilers** [YOYOK9/BusinessElite-Plus#34]
#### Issue Description:
Uncrafting grinder has broken outputs

#### What Happens:
When uncrafting an item, it only gives one item type, but it floods the output with exceeding amounts of the item, up to a stack

#### What you Expected to happen:
It gives out the recipe

#### Steps to Reproduce in detail:
1. Place item to uncraft in input.
2. Wait
3. Get loads of output
...
#### Does it happen on the Server: (yes or no)
[x ] Yes

[ ] No

#### Pack Versions: Ex(V165.1.0.1)
165.1.0.7

#### Mods Effected: Ex: (Quark, AE2, ect) 
Uncrafting grinder from Cyclic
Basically any mod that has normal crafted items.

#### Your most recent log file where the issue was present or a screenshot of the issue:  ex: (log folder or crash reports or screenshot folder)
Does not crash the server or give a log fire

[Pastebin/Gist/Pictures LINK here]
![image](https://user-images.githubusercontent.com/81107385/111915769-4edb2900-8a4e-11eb-9998-cf52acafbf54.png)

**Comment by YOYOK9** [comment]
Seems to be fixed in my current update
https://puu.sh/Hrbhi/9076400cf8.gif
